appmap/test/test_django.py

```diff
@@ -150,6 +155,27 @@ def test_included_view(client, events):
         'normalized_path_info': '/post/included/{username}'
     })
 
+@pytest.mark.appmap_enabled
+def test_exception(client, events, monkeypatch):
+    def raise_on_call(*args):
+        raise RuntimeError('An error')
+    monkeypatch.setattr(django.core.handlers.exception, 'response_for_exception', raise_on_call)
+
+    with pytest.raises(RuntimeError):
+        client.get('/exception')
+
+    assert events[0].http_server_request.items() >= {
+        'request_method': 'GET',
+        'path_info': '/exception',
+        'protocol': 'HTTP/1.1'
+    }.items()
+
+    assert events[1].parent_id == events[0].id
+    assert events[1].exceptions[0].items() >= {
+            'class': 'builtins.RuntimeError',
+            'message': 'An error'
+    }.items()
```

Comment:
You can use DictIncluding now:
```suggestion
    assert events[0].http_server_request == DictIncluding({
        'request_method': 'GET',
        'path_info': '/exception',
        'protocol': 'HTTP/1.1'
    })

    assert events[1].parent_id == events[0].id
    assert events[1].exceptions == [DictIncluding({
            'class': 'builtins.RuntimeError',
            'message': 'An error'
    })]
```